src/tape/ensemble.py

```diff
@@ -1002,49 +1126,26 @@ def from_parquet(
                     columns.append(col)
 
         # Read in the source parquet file(s)
-        self._source = dd.read_parquet(
-            source_file, index=self._id_col, columns=columns, split_row_groups=True
-        )
-
-        if object_file:  # read from parquet files
-            # Read in the object file(s)
-            self._object = dd.read_parquet(object_file, index=self._id_col, split_row_groups=True)
-
-            if self._nobs_band_cols is None:
-                # sets empty nobs cols in object
-                unq_filters = np.unique(self._source[self._band_col])
-                self._nobs_band_cols = [f"nobs_{filt}" for filt in unq_filters]
-                for col in self._nobs_band_cols:
-                    self._object[col] = np.nan
-
-            # Handle nobs_total column
-            if self._nobs_tot_col is None:
-                self._object["nobs_total"] = np.nan
-                self._nobs_tot_col = "nobs_total"
-
-            # Optionally sync the tables, recalculates nobs columns
-            if sync_tables:
-                self._source_dirty = True
-                self._object_dirty = True
-                self._sync_tables()
-
-        else:  # generate object table from source
-            self._object = self._generate_object_table()
-            self._nobs_bands = [col for col in list(self._object.columns) if col != self._nobs_tot_col]
+        source = dd.read_parquet(source_file, index=self._id_col, columns=columns, split_row_groups=True)
 
         # Generate a provenance column if not provided
         if self._provenance_col is None:
-            self._source["provenance"] = self._source.apply(
-                lambda x: provenance_label, axis=1, meta=pd.Series(name="provenance", dtype=str)
-            )
+            source["provenance"] = provenance_label
```

Comment:
What is provenance, do we need it for pandas / dask dataframe inputs?

Reply:
In the case of multi-survey data, the "provenance" column is meant to determine which rows correspond to which survey.

My thoughts are that the pandas/dask dataframe case should be close to the `from_source_dict` where we currently don't populate this for the user. Being a lower-level interface users can always populate the field themselves if needed.